ccmlib/node.py

```diff
@@ -863,12 +863,15 @@ def run_cli(self, cmds=None, show_output=False, cli_options=[]):
                         print(log, end='')
                     i = i + 1
 
-    def run_cqlsh(self, cmds=None, show_output=False, cqlsh_options=None, return_output=False, timeout=600):
+    def run_cqlsh(self, cmds=None, show_output=False, cqlsh_options=None, return_output=False, timeout=600, env=None):
         cqlsh_options = cqlsh_options or []
         cqlsh = self.get_tool('cqlsh')
         if not isinstance(cqlsh, list):
             cqlsh = [cqlsh]
-        env = self.get_env()
+        if env:
+            env = self.get_env().update(env)
+        else:
+            env = self.get_env()
```

Comment:
```suggestion
        env = self.get_env()
        if extra_env is not None:
            env.update(extra_env)
        return env
```
        else:
            env = self.get_env()
```

Reply:
done